products/sle/main.pm

```diff
@@ -592,6 +592,9 @@ sub load_consoletests() {
             loadtest "rt/kmp_modules";
         }
         loadtest "console/consoletest_setup";
+        if (get_var("LOCK_PACKAGE")) {
+            loadtest "console/check_locked_package";
+        }
```

Comment:
openSUSE only has offline migration, I will consider to include it when adding sle offline migration.